lib/components/Tabs/stories.tsx

```diff
@@ -22,4 +25,22 @@ storiesOf('Components|Tabs', module)
 			<Tab title={'A'}>a</Tab>
 			<Tab title={'B'}>b</Tab>
 		</Tabs>
+	))
+	.add('with number', () => (
+		<Tabs>
+			<Tab title="Leads" indication={getRandomInt(6, 10)}>
+				Content a
+			</Tab>
+			<Tab title="Claimed" indication={getRandomInt(1, 3)}>
```

Comment:
Isn't randomising these numbers going to cause unnecessary chromatic whinging?

Reply:
:100: